install/automationservicebroker/deploy-automation-broker-apb.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: automation-broker-apb
+objects:
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: automation-broker-apb
+    namespace: "${NAMESPACE}"
+
+- apiVersion: rbac.authorization.k8s.io/v1beta1
+  kind: ClusterRoleBinding
+  metadata:
+    name: automation-broker-apb
+  roleRef:
+    name: cluster-admin
+    kind: ClusterRole
+    apiGroup: rbac.authorization.k8s.io
+  subjects:
+  - kind: ServiceAccount
+    name: automation-broker-apb
+    namespace: "${NAMESPACE}"
+
+- apiVersion: batch/v1
+  kind: Job
+  metadata:
+    name: automation-broker-apb
```

Comment:
You may want to specifically run this job in the `${NAMESPACE}` where the `automation-broker-apb` service account will live.

Reply:
It runs in the namespace set here:
https://github.com/openshift/origin/pull/19409/files#diff-0b8cd438a5f3db747099c64b502fe091R12
https://github.com/openshift/origin/pull/19409/files#diff-0b8cd438a5f3db747099c64b502fe091R24